### PR TITLE
chore: Convert tabs to spaces and update CODEOWNERS

### PR DESCRIPTION
**Pull request by lamroger**

### CODE REVIEW

File: CODEOWNERS
Comment (cwetherill-ps) [inline CODEOWNERS:3]: Just future-proofing a little since other product teams will probs be added as global owners as well at some point, Codeowners _should_ follow the same syntax as `.gitignore` files, so I think
```
**/*deployment*.py  @Paperspace/gradient-deployments
**/*model*.py       @Paperspace/gradient-deployments
```
should be valid and give our team more narrowly-scoped ownership to files we specifically care about with the benefit of directory globbing so we don't need to care where in the repo those files exactly live

**Review by cwetherill-ps — APPROVED** [review]
LGTM pending the codeowners nit!

File: CODEOWNERS
Comment (lamroger) [reply CODEOWNERS:3]: Interesting - didn't know github enforced owners like that!

**Pull request merged**
